addons/compat_ace_fire/XEH_PREP.hpp

```diff
@@ -0,0 +1 @@
+[QPATHTOF(functions\fnc_burnSimulation.sqf), QUOTE(DOUBLES(REPLACEMOD,fnc_burnSimulation))] call CBA_fnc_compileFunction;
```

Comment:
what is going on here, why not just use the normal prep macro?

Reply:
Replaces an ace function, using the normal prep would just create a new one not used by ace.